config/src/iroha.rs

```diff
@@ -144,11 +144,8 @@ impl ConfigurationProxy {
                     message: "Torii config should have at least `p2p_addr` provided for sumeragi finalisation",
                 });
             }
-            // Finally, if trusted peers were not supplied, we can fall back to inserting itself as
-            // the only trusted one
-            if sumeragi_proxy.trusted_peers.is_none() {
```

Comment:
Is the check not necessary and `trusted_peers` is always `None`?

Reply:
it's not always `None` check the `insert_self_as_trusted_peers`

Reply:
oh, I missed the changes in the method.
Looked at my local version